packages/mobile/src/notifications.ts

```diff
@@ -89,17 +89,15 @@ class PushNotifications {
     }
   }
 
+  async hasPermission(): Promise<boolean> {
+    return await Notifications.isRegisteredForRemoteNotifications()
+  }
+
   requestPermission() {
     isRegistering = true
     Notifications.registerRemoteNotifications()
   }
 
-  async checkPermission(
```

Comment:
this doesnt seem to be called anywhere and does the same thing as isRegisteredForRemoteNotifications but ios-specific

Reply:
Good catch! Looks fine to delete for sure